aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -259,6 +294,35 @@ Tensor& mul_(Tensor& self, const Tensor& other) {
   return native::mul_out(self, self, other);
 }
 
+Tensor mul(const Tensor& self, Scalar other) {
+  return native::mul(self, wrapped_scalar_tensor(other));
```

Comment:
is this correct?  Won't this skip dispatch which means you can't dispatch to the sparse variant?

python might avoid this by always calling the tensor overload, but maybe it's an issue in C++?

Reply:
You're probably right. This is existing code that I just moved next to the other mul functions for consistency. I've made a note to review this behavior.